### ISSUE TITLE
Whole Logseq window disappeared

### ISSUE THREAD

**Issue by gpytel** [yoyurec/logseq-awesome-ui#79]
**Info:**
- OS: MacOS Sonoma
- Logseq ver: latest
- Plugin ver: latest
- Theme name: 
I don't remember, and I can't check because Logseq disappeared!

- [ ] ⚠ Do i have issue with DISABLED my `custom.css`?
Can't check because Logseq disappeared!

**Description:**
I installed the plugin and it changed Logseq in the background when I was still in the Marketplace. I went to the main window, and I noticed there is a small gap on top. My thinking was that the windows is not in full screen, so I double-clicked, to bring it either to full screen, or small screen. What was my surprise when it went away completely!

Now the software icon has a dot for a software that is running, but there is no window. If I use Misson Control, it does give me a shade, but I cannot get it on the screen :-( I restarted Logseq, I restarted my laptop, but that didn't help!

**To Reproduce:**
 Try doing what I did...

**Expected behavior:**
 ...

**Screenshots**

<img width="491" alt="Screenshot 2024-01-22 at 9 25 26 PM" src="https://github.com/yoyurec/logseq-awesome-ui/assets/28045418/0783e2df-060b-4249-9e0a-88abd53e61e5">

<img width="491" alt="Screenshot 2024-01-22 at 9 25 33 PM" src="https://github.com/yoyurec/logseq-awesome-ui/assets/28045418/34298980-dc45-4ba0-bb03-59519e429171">

**Attach files**
Add (via drag-n-drop) .zip with `.md` files (if it needs) to help with bugfixing.

Please help me get the app back!

**Comment by gpytel** [comment]
Ugh, solved it myself! I moved the Logseq (using the top bar menu, because that was still available) to my external monitor and I suddenly saw a drop, the smallest ever drop on the second screen. When I hovered over it, I got the resize arrows! Yay! I managed to resize the smallest window and have my Logseq back! However - I uninstalled the plugin right away because I lost the trust other functions will not make it disappear again...

**Comment by gpytel** [comment]
Ugh, solved it myself! I moved the Logseq (using the top bar menu, because that was still available) to my external monitor and I suddenly saw a drop, the smallest ever drop on the second screen. When I hovered over it, I got the resize arrows! Yay! I managed to resize the smallest window and have my Logseq back! However - I uninstalled the plugin right away because I lost the trust other functions will not make it disappear again...
